pmlc/rt/runtime.h

```diff
@@ -46,13 +47,40 @@ class Device {
 // evaluating a PlaidML program.  These are typically single-instanced within a
 // process, constructed once as needed (typically well after process
 // initialization) and then held onto by the global runtime registry.
+//
+// N.B. The Runtime lifecycle is a little unusual.
```

Comment:
Can we just say:

```
The Runtime Lifecycle:
```

And then let the reader decide if they think it's unusual or not.

Reply:
Okay.  But it _is_ unusual: in this design, `Runtime` is not a normal object, initialized to a usable state in its constructor; there's this extra step you need to know to perform, and that is definitely worth calling out in big bold flashing marquee text.

Reply:
That's subjective, please leave that out of code comments :)